### PR TITLE
Fix template changes dark theme in tmux v2.9

### LINKED ISSUE #6: Doesn't work on Tmux 2.9
```
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:9: invalid option: window-status-bg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:10: invalid option: window-status-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:12: invalid option: window-status-activity-bg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:13: invalid option: window-status-activity-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:16: invalid option: window-status-current-bg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:17: invalid option: window-status-current-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:20: invalid option: pane-active-border-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:21: invalid option: pane-border-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:24: invalid option: message-bg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:25: invalid option: message-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:28: invalid option: message-command-bg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:29: invalid option: message-command-fg
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:43: invalid option: status-attr
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:45: invalid option: status-left-attr
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:47: invalid option: status-right-attr
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:49: invalid option: window-status-activity-attr
/home/andrea/.config/tmux/tmux-gruvbox-dark.conf:50: invalid option: window-status-attr
```

I think the format has changed:
https://fossies.org/diffs/tmux/2.8_vs_2.9/CHANGES-diff.html

### PR DESCRIPTION
**Pull request by egel**
Close #6 

- Fix problems with breaking changes with the new release of tmux v2.9
- Simplify the code with inline -style syntax

### CODE REVIEW

**Comment by egel** [comment]
Current result in my OSX Mojave 10.14.4 looks following:

Screenshot 2019-04-30 at 00 50 26

File: tmux-gruvbox-dark.conf [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
 
 # message infos
-set-option -g message-bg colour239 #bg2
-set-option -g message-fg colour223 #fg1
+set-option -g message-style bg=colour239,fg=colour223 # bg=bg2, fg=fg1
 
 # writting commands inactive
-set-option -g message-command-bg colour239 #fg3
-set-option -g message-command-fg colour223 #bg1
+set-option -g message-command-style bg=colour239, fg=colour223 # bg=fg3, fg=bg1
```
Comment (polyzen) [inline tmux-gruvbox-dark.conf]:
```suggestion
set-option -g message-command-style bg=colour239,fg=colour223 # bg=fg3, fg=bg1
```

Seems there's excess whitespace.

File: tmux-gruvbox-dark.conf [hunk 2]
```diff
@@ -36,23 +30,21 @@ set-option -g display-panes-colour colour237 #bg1
 set-window-option -g clock-mode-colour colour109 #blue
 
 # bell
-set-window-option -g window-status-bell-style fg=colour235,bg=colour167 #bg, red
-
+set-window-option -g window-status-bell-style bg=colour167,fg=colour235 # bg=red, fg=bg
 
 ## Theme settings mixed with colors (unfortunately, but there is no cleaner way)
-set-option -g status-attr "none"
+set-option -g status-style attr=none
```
Comment (polyzen) [inline tmux-gruvbox-dark.conf]: `Bad style: attr=none`

File: tmux-gruvbox-dark.conf [hunk 3]
```diff
@@ -36,23 +30,21 @@ set-option -g display-panes-colour colour237 #bg1
...
 set-option -g status-justify "left"
-set-option -g status-left-attr "none"
+set-option -g status-left-style attr=none
 set-option -g status-left-length "80"
-set-option -g status-right-attr "none"
+set-option -g status-right-status attr="none"
```
Comment (polyzen) [inline tmux-gruvbox-dark.conf]: `Invalid option: status-right-status`

Comment (koalp) [reply tmux-gruvbox-dark.conf]: It should be `status-right-style` I think.

```suggestion
set-option -g status-right-style none
```

File: tmux-gruvbox-dark.conf (hunk 2 above)
Comment (koalp) [reply tmux-gruvbox-dark.conf]: According to the man, `attr` is not a valid style but `none` is.
```suggestion
set-option -g status-style none
```

**Comment by AndreaOrru** [comment]
`status-style none` does not result in the expected styling:
image
The background of the bar is black, and it should be grey.
However, if I just comment all the lines with `attr=none`, it works as expected.

File: tmux-gruvbox-dark.conf (hunk 2 above)
Comment (egel) [reply tmux-gruvbox-dark.conf]: You're right, I've accepted your change.

**Comment by egel** [comment]
> `status-style none` does not result in the expected styling:
> image
> The background of the bar is black, and it should be grey.
> However, if I just comment all the lines with `attr=none`, it works as expected.

I think that happened because I've changed `status-style` 2 times in the file. The last one completely disabled the changed, but this worked IMHO only due to this option (attr=none) has been disabled in the newer versions of tmux.

**Comment by egel** [comment]
I've updated the configuration and I've checked again on: 

1. OSX Mojave 10.14.4; tmux 2.9

    Screenshot 2019-05-02 at 00 15 07

2. Ubuntu 16.04.5; tmux 2.1

    Screenshot 2019-05-02 at 00 12 38

Please @AndreaOrru check again if updated config works for you. If yes, I will be going to merge this PR.

**Comment by AndreaOrru** [comment]
It works ;)

**Comment by egel** [comment]
Great, then I will merge this asap. Thank you @AndreaOrru, @koalp, and @polyzen for your contribution 🥳👍

**Pull request merged**